homeassistant/components/config/config_entries.py

```diff
@@ -284,6 +285,28 @@ def get_entry(
     return entry
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        "type": "config_entries/get",
+        "entry_id": str,
+    }
+)
+@websocket_api.async_response
+async def config_entry_get(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Update config entry."""
+    entry = get_entry(hass, connection, msg["entry_id"], msg["id"])
```

Comment:
I'm seeing errors in a new integration I'm developing related to this change.
```
Error handling message: Unknown error (unknown_error) test_user from ::1 (Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36)
Traceback (most recent call last):
  File "/home/scott/code/core/homeassistant/components/websocket_api/decorators.py", line 26, in _handle_async_response
    await func(hass, connection, msg)
  File "/home/scott/code/core/homeassistant/components/config/config_entries.py", line 302, in config_entry_get
    entry = get_entry(hass, connection, msg["entry_id"], msg["id"])
KeyError: 'entry_id'
```
The error occurs when going to the devices page for the integration. E.g:
http://localhost:8123/config/devices/dashboard?historyBack=1&config_entry=123abc

Reply:
Are you using a matching frontend branch? (dev)

Reply:
I was thinking that was the problem since it looked frontend related, but doesn't core set the version of the frontend it needs? I know I've seen PRs updating the version (e.g. #92508)? Meaning, if this PR's changes needed a specific version of the frontend, it or a previous PR should have bumped the frontend to that version, no?

Reply:
Sometimes we sync them up, but generally only before beta or stable releases. During dev, they are decoupled.